diskqueue.go

```diff
@@ -301,6 +335,10 @@ func (d *diskQueue) readOne() ([]byte, error) {
 			stat, err := d.readFile.Stat()
 			if err == nil {
 				d.maxBytesPerFileRead = stat.Size()
+				if d.diskLimitFeatIsOn {
+					// last 8 bytes are reserved for the number of messages in this file
+					d.maxBytesPerFileRead -= 8
```

Comment:
For readability, I see 8 is used in several places. You can make it as a constant.

Reply:
Sounds good, how does numFileMsgsBytes sound?